internal/acceptance/openstack/identity/v3/roles_test.go

```diff
@@ -94,19 +94,16 @@ func TestRolesCRUD(t *testing.T) {
 
 	th.AssertEquals(t, found, true)
 
+	description := ""
```

Comment:
Looks like we try to update description from an empty string to... an empty string. If we want to test that we can unset the description (if that was your intention) we should set the original value to something else.